sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Security/Ucfg/KnownMethodId.cs

```diff
@@ -0,0 +1,29 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2018 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarAnalyzer.Security.Ucfg
+{
+    public static class KnownMethodId
```

Comment:
What are these - names as specified in the UCFG? If so, _UCFGMethodId_ / _UCFGMethodKind_ would be clearer.
_Id_ is potentially confusing since there is also a _MethodIdProvider_ that does something quite different. This enum seems more like the Roslyn _SyntaxKind_.